### PR TITLE
Pass in Current Participant

### PR DESCRIPTION
**Pull request by michaelwehrley**
* Pass in current participant to `HomeCtrl`
* Update controller specs

[Finishes: #118139125]

### CODE REVIEW

File: app/assets/javascripts/social_networking/controllers/home-controller.js [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
   // Provides access to the feed and its items.
   function HomeCtrl(OnYourMindResource, CommentResource, LikeResource,
```
Comment (ericcf) [inline app/assets/javascripts/social_networking/controllers/home-controller.js]: Wow, that's a lot of dependencies being injected ;-)

File: app/assets/javascripts/social_networking/controllers/home-controller.js [hunk 2]
```diff
@@ -265,14 +266,14 @@
 
   function participantHasLiked(controller, item) {
     return (controller._findLikes(item.likes, {
-      participantId: controller._participantId
-    }) || []).length !== 0;
+      participantId: controller._currentParticipantId
+    }) || []).length;
```
Comment (ericcf) [inline app/assets/javascripts/social_networking/controllers/home-controller.js]: `truthy`!

**Comment by ericcf** [comment]
In the long run, this could use refactoring.

**Comment by eschlange** [comment]
@ericcf I agree, this controller and home page is a beast. I added a refactor story to TFD Maintenance: https://www.pivotaltracker.com/story/show/118291163

File: app/assets/javascripts/social_networking/controllers/home-controller.js (hunk 2 above)
Comment (michaelwehrley) [inline app/assets/javascripts/social_networking/controllers/home-controller.js]: haha, yep.

**Pull request merged**